### PR TITLE
Add env var to ignore bad Pub/Sub messages.

### PR DESCRIPTION
**Pull request by LOZORD**
These bad messages can get stuck and may introduce thundering herds
of traffic, which may cause processing latency (#71 ).

Setting the environment variable `IGNORE_BAD_MESSAGES` to a non-empty
string will trigger this ignoring/swallowing behavior.

### CODE REVIEW

**Review by navierula — APPROVED** [review]

File: lib/notifiers/notifiers.go [hunk 1]
```diff
@@ -216,10 +217,12 @@ func Main(notifier Notifier) error {
 		return fmt.Errorf("failed to call SetUp on notifier: %w", err)
 	}
 
+	_, ignoreBadMessages := GetEnv("IGNORE_BAD_MESSAGES")
```
Comment (Philmod) [inline lib/notifiers/notifiers.go:220]: What does happen when this env variable doesn't exist?
And how does it know it's a boolean?

**Review by Saaarah — APPROVED** [review]
LGTM

File: lib/notifiers/notifiers.go (hunk 1 above)
Comment (LOZORD) [reply lib/notifiers/notifiers.go:220]: `GetEnv` returns (string, bool) -- the bool is true iff the string is nonempty. I think we can just document this as "If you want to enable this feature, just set `IGNORE_BAD_MESSAGES=true`". Does that sound ok?

Comment (Philmod) [reply lib/notifiers/notifiers.go:220]: So if `IGNORE_BAD_MESSAGES = false`, it won't do what is expected by the user?

Comment (LOZORD) [reply lib/notifiers/notifiers.go:220]: They just have to unset it (`IGNORE_BAD_MESSAGES=` or remove it from the environment all together).

**Pull request merged**